### DISCUSSION TITLE
endwise for languages like ruby

### DISCUSSION THREAD

**Discussion by otavioschwanck** [helix-editor/helix#10201] [General]
Something like, when i type def, it adds the end automatically.  

```ruby
def something
end # end added autpmatically after typed def<space>
```

**Comment by the-mikedavis** [comment]
I'm not sure if the ruby language server provides this but there's a similar feature provided by ElixirLS based on snippets (see https://github.com/helix-editor/helix/issues/3899)

If it's not based on something language-agnostic like snippets, this kind of language-specific feature would belong to the plugin-space rather than core.
